clang/include/clang/AST/Type.h

```diff
@@ -4033,12 +4033,27 @@ class FunctionType : public Type {
     SME_NormalFunction = 0,
     SME_PStateSMEnabledMask = 1 << 0,
     SME_PStateSMCompatibleMask = 1 << 1,
-    SME_PStateZASharedMask = 1 << 2,
-    SME_PStateZAPreservedMask = 1 << 3,
-    SME_AttributeMask = 0b111'111 // We only support maximum 6 bits because of the
-                                  // bitmask in FunctionTypeExtraBitfields.
+
+    // Describes the value of the state using ArmStateValue.
+    SME_PstateZAShift = 2,
```

Comment:
It might make sense for these to be just `SME_ZA…`, rather than `SME_PstateZA…`, since they're now specifically about the contents of ZA (rather than PSTATE.ZA).